scraper/util.py

```diff
@@ -148,7 +148,7 @@ def compute_labor_hours(sloc, month_hours="cocomo_book"):
     The intention is to use the COCOMO II model to compute this value.
 
     References:
-    - https://csse.usc.edu/tools/cocomoii.php
+    - http://csse.usc.edu/tools
```

Comment:
I'm having trouble with this link loading... does this one also need to be the softwarecost.org site as in the code?

Reply:
This loads the USC site for me (which is where I found the softwarecost.org url)  but if it doesn’t work for you I’ll make it the same softwarecost.org URL as the code.

Reply:
Ah ok, nevermind, it just takes a long time for me to load (not sure why), we can leave this. 👍